src/users/users.service.ts

```diff
@@ -71,23 +73,31 @@ export class UsersService {
     };
   }
 
-  async findByEmail(email: string): Promise<User> {
-    this.log.debug(`Called findByEmail ${email}`);
-    const user = await this.usersRepository.findOne({ email });
+  async findUser(query: string): Promise<User> {
```

Comment:
You should declare a separate method and not mix them (e.g. `findById` and `findByEmail`)

Reply:
@derevnjuk AppSecs requested method which searching by email and id the same time

Reply:
https://github.com/NeuraLegion/brokencrystals/pull/204#discussion_r960948495